build.sh

```diff
@@ -209,25 +209,33 @@ isMSBuildOnNETCoreSupported()
     # This needs to be updated alongwith corresponding changes to netci.groovy.
     __isMSBuildOnNETCoreSupported=0
```

Comment:
Does it make more since to invert this default and assume it is supported and detect failure in some other way?